src/components/users/UserForm.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { Formik, Form } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { TextField } from '@material-ui/core';
+import { Typography } from '@mui/material';
+import { useNavigate, useLocation } from 'react-router-dom';
+import Button from '@mui/material/Button';
+import ToastContext from "../../context/ToastContext";
+import UserValidationSchema from '../../pages/users/UserValidationSchema';
+import { reset } from '../../context/redux/actions';
+import store from '../../context/redux/store';
+
+export default function UserForm() {
+    const [singleUser, setSingleUser] = useState([]);
+    const { t } = useTranslation();
+    const [isLoading, setIsLoading] = useState(false);
+    const { setToastState } = useContext(ToastContext);
+    const navigate = useNavigate();
+    const location = useLocation();
+
+    const getUserId = () => {
+        return (location.pathname).slice(7);
```

Comment:
7 is highly specific to the position. split the strings by "/" and take the last element of that array

Reply:
changed the function to get the last element of the array